docs/book/content/theory/firms.md

```diff
@@ -45,7 +55,64 @@ The production side of the `OG-Core` model is populated by a unit measure of ide
 
   ```{math}
   :label: EqFirmFOC_K
-    r_t = (1 - \tau^{corp})(Z_t)^\frac{\varepsilon-1}{\varepsilon}\left[\gamma\frac{Y_t}{K_t}\right]^\frac{1}{\varepsilon} - \delta + \tau^{corp}\delta^\tau \quad\forall t
+    r_t = (1 - \tau^{corp}_t)(Z_t)^\frac{\varepsilon-1}{\varepsilon}\left[\gamma\frac{Y_t}{K_t}\right]^\frac{1}{\varepsilon} - \delta + \tau^{corp}\delta^\tau \quad\forall t
   ```
 
   Note that the presence of the public capital good creates economic rents. However, given perfect competition, any economic profits will be competed away. For this reason, the optimality condition for capital demand {eq}`EqFirmFOC_K` is only affected by public capital $K_{g,t}$ through the $Y_t$ term.
+
+(EqFirmsPosProfits)=
+## Positive Profits from Government Infrastructure Investment
+
+  The CES production function in {eq}`EqFirmsCESprodfun` exhibits constant returns to scale (CRS). A feature of CRS production functions is that total revenue is a sum of the revenue attributed to each factor,
```

Comment:
maybe add (to note that what is below isa simplification without taxes): `... to each factor, for example, without taxes we have:`

Reply:
@jdebacker. This expression is derived purely from the production function. Taxes are not involved with its derivation. So maybe I could change the wording in the last sentence to "...gross revenue is a sum of the gross revenue attributed to each factor of production." However, I suspect there is better wording that we could take from a micro textbook. I looked through Varian, Nicholson, Mas-Colell and couldn't find anything.

Reply:
@jdebacker. I updated this sentence in commit [0bd5b38](https://github.com/jdebacker/OG-Core/pull/24/commits/0bd5b38cbfb03405e4a621b24d4fd0853a8ceda3) to read, "...gross revenue $Y_t$ is a sum of the gross revenue attributed to each factor of production,".